client/components/vote-escrow/LockupForm.tsx

```diff
@@ -302,8 +302,8 @@ const LockupForm: FunctionComponent<LockupFormProps> = ({ existingLockup }) => {
         transaction = await contracts.OgvStaking["extend(uint256,uint256)"](
           existingLockup.lockupId,
           duration,
-          { gasLimit: 1000000 }
-        ); // @TODO maybe set this to lower
+          { gasLimit: 154727 }
```

Comment:
This is tight.
Tom did an extend on mainnet
https://etherscan.io/tx/0x3f0ba0620459079f8b2d55c380f2409aa876e8776c2b481281af09061ebebb3a

It used 149,604 

How about 150k*1.3 = 195k gas?